concepts/observability-strategy/opentelemetry-operator/README.md

```diff
@@ -0,0 +1,89 @@
+
+Re-inventing the wheel should be avoided, so the PoC discusses the impact on introducing the operator in irder to provide the mentioned features to the users.
+
+## Scope
+Introducing the operator to the stack either as underlying technology only or actual feature, the effect will be that the CRDs will be visible to runtime users and users can independently use them. Trying to isolate them will not be of success as previous experience with the prometheus-operator shows that this will only cause conflicts with potential operator installations provided by the user. Meaning of: restricting the operator usage in the runtime AND blocking the installation of a custom operator installation is not an option as users cannot leverage the power of it at all. Running a restricted operator for internal usage and allowing to install a custom operator is resulting in many potential conflicts like race-reconcilations on the same resources or webhook conflicts (as custom operator is not excluding system namespaces).
+Conclusion: either the operator gets used as underlying technology for the kyma abstractions and instrumentation support AND is accessible for custom scenarios for users. Or the operator gets not used. No other option.
+
+## Proposal
+
+- The existing `telemetry-operator` manages an opentelemetry-operator additionally.
+- The current abstraction `TracePipeline` does not reveal into an actual deployment but an `OpenTelemetryCollector` resource configuration. The configuration gets reconciled and with that cannot be influenced by the user. The `opentelemetry-collector` will resolve the configuration into an actual deployment. That part is still optional and the indirection via the opentelemetry-operator is not mandatory. It will be mainly beneficial for the user to copy-over sich configuration in order to realize a custom trace-collector
+- The telemetry-operator reconciles `Instrumentation` resources which can be used by the user instantly for auto-instrumentation and injection of the `trace-collector` reference
+- The user can specify custom `OpenTelemetryCollector` resource for custom exporter handling or any other activities likes running a sidecar.
+- The user can specify custom `Instrumentation` for env injection of custom collectors.
+
+![Architecture](./assets/architecture.drawio.svg)
+
+## POC
+
+### Prerequisites
+- Have a Kyma cluster in version >= 2.10
+- Have a TracePipeline configured with a valid output
+
+
+### Installation
+Install the Otel Operator using helm
```

Comment:
```suggestion
1. Install the Otel Operator using Helm:
```
Ideally, we should add blank spaces for proper indentation of the commands below the instruction, but you asked for a rough review, so I'm gonna ignore that :P